### ISSUE TITLE
Please add this feature

### ISSUE THREAD

**Issue by asdf337** [reblws/tab-search#71]
So we can search for words and items INSIDE the tab, not just in the URL itself. My URLS all look the same and I have 200 of them, I need to search whats inside the URLS. Thanks, hope you can add this for us :)

**Comment by bubblefoil** [comment]
1. Can you edit the title to be anything useful? This just says: "Don't even bother reading".
2. [Is this other add-on what you are looking for?](https://addons.mozilla.org/en-US/firefox/addon/search-multi-tabs/)

**Comment by asdf337** [comment]
Yeah I ended up using that one, the title isnt the best but my description inside is decent enough for you to understand it.

**Comment by bubblefoil** [comment]
That is not the point. A descriptive title is useful in the list of issues. There are issue labels exactly for this purpose, like `Feature request`. Please, keep in mind that you request someone else to implement a feature for you and others. For free. Typing those two sentences is all the effort on your side. You could at least try just a little bit harder ;-)

**Comment by asdf337** [comment]
You know the way you wrote to me in the first place wasnt the best either,
all you needed to say was "could you please provide some more information
in the title?" instead of being rude and saying "Dont even bother reading
this", which is exactly what I'll do with such
rude answers ;-)

**Comment by bubblefoil** [comment]
Ok, I could have tried to formulate the answer more politely. I would argue the first one was "rude", though.
BTW how do you know an answer is rude without reading it?

Please, somebody close this issue, it brings out the bad child part of us.

**Comment by asdf337** [comment]
I added "this" on the end cause its should have been there and now you tell
me I didnt even read it? There is no reason for you to argue if it was rude
or not cause you seem like a narcissistic piece of shit anyway, you would
never admit your own faults :) I'll close this for ya, BYE BYE : > )

**Comment by bubblefoil** [comment]
You seem like a nice guy, I am sure we would resolve this dispute over a glass of beer. Too bad this is the internet. I hate this place.
Thanks for the "this", I appreciate every opportunity to improve my language. Cause I like myself better when I know stuff and correct other people.
PS: I figured you read it, I thought that the conditional with present simple is meant for the future. Another language lesson learnt :-)
